src/schema.py

```diff
@@ -144,9 +144,10 @@ def resolve_account_info(self, info, session_id=None):
       location = txn['locationName'].rsplit(' ', 1)[0]
       # removes the register numbers at the end of the string by taking the substring up until
       # the last space (right before the number)
+      name = LOCATION_NAMES.get(location, {'name': location})['name']
```

Comment:
this code isn't intuitive, probably more readable to do:
```python
name = LOCATION_NAMES['location']['name'] if location in LOCATION_NAMES else location
```

Also, `LOCATION_NAMES` isn't a good name for this constant anymore!